content/fr/blog/sample-letter-gdpr-direct-marketing-objection/index.md

```diff
@@ -9,7 +9,27 @@
     "license": "cc0-10"
 }
 
-Conformément à l'Art. 21, alinéa 2 du RGPD, tu as le droit de t'opposer à l'utilisation de tes données personnelles à des fins de prospection.
+La publicité semble incontournable aujourd'hui: dès que vous ouvrez votre boîte aux lettre ou email, vous recevez souvent une série de lettres ou d'emails, d'entreprises qui veulent promouvoir leurs produits. Souvent, elles vous sont même adressées personnellement.
+
+Pour le fondement juridique, ces entreprises invoquent fréquemment leur "intérêt légitime" ou votre consentement, souvent enfoui très profondément dans un contrat sur un sujet totalement différent.
+
+{{< featuredImg alt="Modèle de lettre pour les objections au marketing direct" >}}
+
+Par chance, vous n'avez pas à supporter cela. Conformément au Règlement Général de la Protection des Données (RGPD), vous avez le droit de vous opposer au marketing direct. Cela est clairement inscrit dans l'Article 21 alinéa 2 du RGPD. Vous pouvez aussi faire valoir ce droit si vous avez déjà donné votre accord.
+
+Le RGPD vous donne d'autres droits dont vous pouvez prendre connaissance dans notre {{< link slug="blog/tes-droits-rgpd/" text="article sur les droits accordés par le RGPD" >}}
```

Comment:
Could you try it with the English slug/name of the directory (`blog/your-gdpr-rights`)? I think it expects that.